Userland/Libraries/LibAudio/RIFFTypes.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2018-2023, the SerenityOS developers.
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/FixedArray.h>
+#include <AK/Forward.h>
+#include <AK/MemoryStream.h>
+#include <AK/Types.h>
+
+// RIFF-specific type definitions necessary for handling WAVE files.
+namespace Audio::RIFF {
+
+static constexpr StringView const riff_magic = "RIFF"sv;
+static constexpr StringView const wave_subformat_id = "WAVE"sv;
+static constexpr StringView const data_chunk_id = "data"sv;
+static constexpr StringView const list_chunk_id = "LIST"sv;
+static constexpr StringView const info_chunk_id = "INFO"sv;
```

Comment:
I think these two are unused? There's a FIXME already, let's leave it up to the person solving it.

Reply:
:thonk: they should be used by the wav metadata loader